portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,21 @@
   </head>
   <body>
     <div id="content">
-    <h1>My Portfolio</h1>
-    <p>Hi! I'm Lily, a STEP Intern at Google this summer and a rising junior @ Stanford studying CS + English.</p>
-    <img src="/images/flower.jpg" alt="Red Flower">
-    <p>Click here to get a random message:</p>
-    <button onclick="addRandomGreeting()">Hello</button>
-    <div id="greeting-container"></div>
+      <h1>Lily Zhou</h1>
+      <div class="icon-row">
+        <a href="https://www.linkedin.com/in/lily-z-408069122"><img src="/images/linkedin.jpg" width="30" height="30" alt="LinkedIn"></a>
```

Comment:
Could you remove the first `/` in the image `src`s order to make this a relative rather than absolute path? This will make them viewable even if the page isn't being loaded from the top level of a domain name. For example, I can preview your HTML change [on htmlpreview.github.io](https://htmlpreview.github.io/?https://github.com/lilymzhou/stepfolio/blob/1f8c3e71234b8c6ef460cd583334f830049e4a15/portfolio/src/main/webapp/index.html), but the leading `/` causes the browser to look for the image file at `https://htmlpreview.github.io/images/linkedin.jpg` which doesn't load. Making it relative will make the image load*.

\* Because the htmlpreview thing is kinda a weird special-ish case, it sets a `<base href="blahblah">` value in the `<head>`, so it's actually relative to that path rather than the path shown in the browser's address bar. But feel free to not worry about that unless you're really curious about it.